app/src/main/java/blockeq/com/stellarwallet/services/networking/Horizon.kt

```diff
@@ -57,5 +61,64 @@ class Horizon {
             }
 
         }
+
+        class SendTask(private val listener: OnSendPayment, private val destAddress: String,
+                       private val sourceKeyPair: KeyPair,
+                       private val memo: String, private val amount : String) : AsyncTask<Void, Void, ErrorResponse>() {
+
+            override fun doInBackground(vararg params: Void?): ErrorResponse? {
+                val server = Server(PROD_SERVER)
+                val destKeyPair = KeyPair.fromAccountId(destAddress)
+
+                Network.usePublicNetwork()
+
+                try {
+                    // Check if destination account exists
+                    server.accounts().account(destKeyPair)
+                } catch (error : ErrorResponse) {
+                    Log.d(TAG, error.body.toString())
+                    return error
+                }
+
+                try {
```

Comment:
Can this be done in the previous try catch?

Reply:
Yes!